react-components/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,8 @@
+
+export default function CustomButton() {
+  return (
+    <>
```

Comment:
The empty braces aren't necessary if the element only has one root. Although the code works with them there, it's generally considered bad practice to use them when they aren't necessary.

Reply:
<img width="566" alt="Screenshot 2023-03-27 at 4 34 16 PM" src="https://user-images.githubusercontent.com/110208589/228090111-020c7086-ec4a-4728-b223-dc153cd8a225.png">